CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ This app presents the landing page experience for
 landregistry.data.gov.uk, including the SPARQL
 Qonsole
 
+## 1.5.7 - 2021-12-01
```

Comment:
Should there also be a `version.rb` change to match this?

Reply:
Yep. I looked for where that was kept, but brain failed me.